src/SFA.DAS.AssessorService.EpaoImporter/PrintProcessCommand.cs

```diff
@@ -135,49 +135,65 @@ private async Task UploadCertificateDetailsToPinter()
                 var batchLogResponse = await _assessorServiceApi.GetCurrentBatchLog();
 
                 var batchNumber = batchLogResponse.BatchNumber + 1;
-                var certificates = (await _assessorServiceApi.GetCertificatesToBePrinted()).ToList()
-                    .Sanitise(_aggregateLogger);
+                var certificates = (await _assessorServiceApi.GetCertificatesToBePrinted()).ToList();
 
                 if (certificates.Count == 0)
                 {
                     _aggregateLogger.LogInfo("No certificates to process");
                 }
                 else
                 {
-                    var certificateFileName =
-                        $"IFA-Certificate-{DateTime.UtcNow.UtcToTimeZoneTime():MMyy}-{batchNumber.ToString().PadLeft(3, '0')}.json";
-                    var excelFileName = $"IFA-Certificate-{DateTime.UtcNow.UtcToTimeZoneTime()}-{batchNumber.ToString().PadLeft(3, '0')}.xlsx";
+                    _aggregateLogger.LogInfo("Certificates found and processing.");
 
-                    var batchLogRequest = new CreateBatchLogRequest
-                    {
-                        BatchNumber = batchNumber,
-                        FileUploadStartTime = DateTime.UtcNow,
-                        Period = DateTime.UtcNow.UtcToTimeZoneTime().ToString("MMyy"),
-                        BatchCreated = DateTime.UtcNow,
-                        CertificatesFileName = certificateFileName
-                    };
+                    await _assessorServiceApi.UpdateBatchNumberInCertificates(batchNumber, certificates);
```

Comment:
This looks like it will update the batch number in Certificates which are not deemed as 'sanitised'

Reply:
This will not be the case anymore since sanitization has now been re introduced. See my previous comment.